[VDG] [Trivial] Coin control. Fix bad confirmation count

SuperJMN opened the pull request:
Calculation was wrong. This showed pockets as being unconfirmed. Not it's fixed.

MarnixCroes reviewed (dismissed):
tACK

ichthus1604 reviewed (approved):
utACK. Code looks good.

soosr reviewed (commented):
I think the number of variables can be decreased (posting all as I removed variables and also moved a property):

```c#
var unconfirmedCount = pocket.Coins.Count(x => !x.Confirmed);
IsConfirmed = pocket.Coins.All(x => x.Confirmed);
ConfirmationStatus = IsConfirmed ? "All coins are confirmed" : $"{unconfirmedCount} coins are waiting for confirmation";
IsBanned = pocket.Coins.Any(x => x.IsBanned);
BannedUntilUtcToolTip = IsBanned ? "Some coins can't participate in coinjoin" : null;
Amount = pocket.Amount;
IsCoinjoining = pocket.Coins.Any(x => x.CoinJoinInProgress);
AnonymityScore = (int) pocket.Coins.Max(x => x.HdPubKey.AnonymitySet);
Labels = pocket.Labels;
Children = pocket.Coins.OrderByDescending(x => x.Amount).Select(coin => new CoinCoinControlItemViewModel(coin)).ToList();
```

SuperJMN commented:
> I think the number of variables can be decreased (posting all as I removed variables and also moved a property):
> 
> ```cs
> var unconfirmedCount = pocket.Coins.Count(x => !x.Confirmed);
> IsConfirmed = unconfirmedCount == 0;
> ConfirmationStatus = IsConfirmed ? "All coins are confirmed" : $"{unconfirmedCount} coins are waiting for confirmation";
> IsBanned = pocket.Coins.Any(x => x.IsBanned);
> BannedUntilUtcToolTip = IsBanned ? "Some coins can't participate in coinjoin" : null;
> Amount = pocket.Amount;
> IsCoinjoining = pocket.Coins.Any(x => x.CoinJoinInProgress);
> AnonymityScore = (int) pocket.Coins.Max(x => x.HdPubKey.AnonymitySet);
> Labels = pocket.Labels;
> Children = pocket.Coins.OrderByDescending(x => x.Amount).Select(coin => new CoinCoinControlItemViewModel(coin)).ToList();
> ```

You're right. Thanks! I've included your modification.

soosr reviewed (approved):
utACK
